src/components/data_search/DatasetSearchTableDisplay.tsx

```diff
@@ -69,7 +132,10 @@ export const DatasetSearchTableDisplay = (props: DatasetSearchTableDisplayProps)
           selected={selected}
           styles={styles}
           tableSize={10}
-          summary={`faceted ${tab.singular} search table`} />
+          summary={`faceted ${tab.singular} search table`}
+          onSort={handleSort}
```

Comment:
Here's an example of where we try to remember the sort selection as part of the user's settings:
https://github.com/DataBiosphere/duos-ui/blob/045a7a36308a6093c78a7dbbf125327ef0224dec/src/components/dar_collection_table/DarCollectionTable.jsx#L349
It would be nice if we could support that, but not a strict requirement for this PR.

Reply:
And here, we try to retrieve it from user settings:
https://github.com/DataBiosphere/duos-ui/blob/045a7a36308a6093c78a7dbbf125327ef0224dec/src/components/dar_collection_table/DarCollectionTable.jsx#L190

Reply:
Oh I should definitely add this for good ux.